codespell_lib/data/dictionary.txt

```diff
@@ -13140,7 +13147,9 @@ frequantly->frequently
 frequences->frequencies
 frequencey->frequency
 frequenct->frequency
```

Comment:
It not one that you added, but perhaps this should be `frequenct->frequency, frequent,`